src/utils/api/lsk/transactions.js

```diff
@@ -167,18 +168,15 @@ export const broadcast = (transaction, network) => new Promise(
  * @returns {Promise<{Low: number, Medium: number, High: number}>} with low,
  * medium and high priority fee options
  */
-export const getTransactionBaseFees = () => (
-  new Promise(async (resolve) => {
-    const fee = 1e3;
-
-    // @todo use real fee estimates
-    resolve({
-      Low: fee,
-      Medium: fee * 2,
-      High: fee * 3,
-    });
-  }));
-
+export const getTransactionBaseFees = network => liskService.getTransactionBaseFees(network)
+  .then((response) => {
+    const { feeEstimatePerByte } = response.data;
+    return {
+      Low: feeEstimatePerByte.low,
+      Medium: feeEstimatePerByte.medium,
+      High: feeEstimatePerByte.high,
+    };
+  });
```

Comment:
This doesn't account for the catch. We decided if this endpoint throws we should show Low = minFee and keep custom fee input enabled.

Reply:
@reyraa is handled at `useTransactionPriorities` using `prioritiesLoadError`